app/views/search/age_groups/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :before_content do %>
+<%= govuk_back_link({
+  text: 'Back',
+  href: root_path(l: params[:l], age_group: params[:age_group]),
```

Comment:
Think you should be able to get these query params from the form object 

```
root_path(l: @age_groups_from.object.l, age_group: @age_groups_form.object.age_group)
```

Reply:
returns `undefined method "object" for nil:NilClass`

Reply:
Woops sorry - 'object' should not be there 

Have pushed a fix 
https://github.com/DFE-Digital/find-teacher-training/pull/1110/commits/f0358c024f696a8f0e726319ed1e831c3383dc65